### ISSUE TITLE
Allow redefining parser, syntax and stringifier PostCSS options

### ISSUE THREAD

**Issue by Dremora** [salsify/ember-css-modules#4]
I've managed to make `ember-css-modules` work with our SASS codebase, however the solution is just one big hack. Would be nice to have a proper way of doing this. Currently `css-modules-loader-core` doesn't accept most of the options listed [here](https://github.com/postcss/postcss/blob/master/docs/api.md#processorprocesscss-opts) but this should be fixed with https://github.com/css-modules/css-modules-loader-core/pull/59.

And, while I'm here, just wanted to say thanks for the addon — I've been waiting for something similar for ages! :laughing:

**Comment by dfreeman** [comment]
Looks like we'll be able to have all kinds of fun once that PR lands!

I'm interested in that support as well – it would be nice for some legacy parts of our codebase, and "does it work with Sass" is probably the most common question I get about this project :)

**Comment by Dremora** [comment]
Just in case someone reading this is interested in using `ember-css-modules` with SASS, this is the solution we've implemented. Caution: it's a big hack, you have to add `postcss` and `postcss-scss` to your `package.json`, and `postcss` should be exactly of the same version as used by `ember-css-modules` (currently `5.0.10`) so that `npm` could flatten them (and you need to be using fairly recent version of npm). And you might need to do `rm -rf node_modules`. I've warned you :)

```javascript
var postcss = require('postcss');
var postCssScssSyntax = require('postcss-scss');

// HACK
var Processor = postcss([]).constructor;
var processorProcess = Processor.prototype.process;
Processor.prototype.process = function (css, opts) {
  opts.syntax = postCssScssSyntax;
  return processorProcess.call(this, css, opts);
}
```

**Comment by dfreeman** [comment]
This is now possible in 0.4.0.
